frameworks/rooch-nursery/sources/multisign_account.move

```diff
@@ -0,0 +1,92 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+/// Bitcoin multisign account module
+module rooch_nursery::multisign_account{
+
+    use std::vector;
+    use std::option;
+    use moveos_std::signer;
+    use moveos_std::object::{Self, Object};
+    use rooch_framework::address_mapping;
+    use rooch_framework::bitcoin_address::{Self, BitcoinAddress};
+
+
+    const ErrorInvalidThreshold: u64 = 1;
+    const ErrorMultiSignAccountNotFound: u64 = 2;
+    const ErrorInvalidParticipant: u64 = 3;
+    const ErrorParticipantMustHasBitcoinAddress: u64 = 4;
+    const ErrorParticipantAlreadyJoined: u64 = 5;
+    const ErrorInvalidPublicKey: u64 = 6;
+
+    struct MultiSignAccountStore has key {
+    }
+
+    struct MultiSignAccount has key {
+        /// The multisign account address
+        bitcoin_address: BitcoinAddress,
+        /// The multisign account rooch address
+        address: address,
+        /// The multisign account threshold
+        threshold: u64,
+        /// The public keys of the multisign account
+        public_keys: vector<vector<u8>>,
+        /// The multisign account participants
+        participants: vector<address>,
+    }
+
+    public(friend) fun genesis_init(){
+        let account_store_obj = object::new_named_object(MultiSignAccountStore{});
+        object::transfer_extend(account_store_obj, @rooch_nursery);
+    }
+
+    fun borrow_mut_account_store() : &mut Object<MultiSignAccountStore> {
+        let object_id = object::named_object_id<MultiSignAccountStore>();
+        object::borrow_mut_object_extend<MultiSignAccountStore>(object_id)
+    }
+
+    fun borrow_account_store() : &Object<MultiSignAccountStore> {
+        let object_id = object::named_object_id<MultiSignAccountStore>();
+        object::borrow_object<MultiSignAccountStore>(object_id)
+    }
+
+    public entry fun create_multi_sign_account(
+        public_keys: vector<vector<u8>>,
+        threshold: u64,
+    ){
+        assert!(vector::length(&public_keys) >= threshold, ErrorInvalidThreshold);
```

Comment:
How many public keys are required for the threshold?

Reply:
threshold <= public_keys length